src/engine/Map.cxx

```diff
@@ -682,24 +682,24 @@ void Map::demolishNode(const std::vector<Point> &isoCoordinates, bool updateNeig
           // get all the occupied nodes and demolish them
           for (auto buildingCoords : getObjectCoords(origCornerPoint, tileID))
```

Comment:
getObjectCoords need to be updated as well
maybe getTileIDCoords

Reply:
changed that